src/Call.tsx

```diff
@@ -897,25 +896,7 @@ export class Call extends EventEmitter {
    *    - Rejects when the message is unable to be sent.
    */
   async sendMessage(message: CallMessage): Promise<OutgoingCallMessage> {
-    if (!(message instanceof CallMessage)) {
-      throw new InvalidArgumentError(
-        'Argument "message" must be instanceof "CallMessage"'
-      );
-    }
-
-    if (
-      typeof message.getContent() === 'undefined' ||
-      message.getContent() === null
-    ) {
-      throw new InvalidArgumentError('"content" is empty');
-    }
-
-    //@ts-ignore
-    if (message.getMessageType().length === 0) {
-      throw new InvalidArgumentError(
-        '"messageType" must be a non-empty string'
-      );
-    }
+    verifySendMessage(message);
```

Comment:
Instead of executing this check when we want to send the message, should we execute it when constructing the message object?

Reply:
This follows the same pattern as the js-sdk https://github.com/twilio/twilio-voice.js/blob/master/lib/twilio/call.ts#L904 

We can, but what if user doesn't send a message? 

@charliesantos any reason for doing the checks before sending message instead of when constructing the message object?

Reply:
It's ok either way. As long as you perform validation before sending it to the server.

Reply:
I strongly prefer doing the check upon construction, that way the developer knows as soon as possible the `CallMessage` is invalid and cannot be sent.

Reply:
done 👍